app/javascript/mastodon/features/hashtag_timeline/components/hashtag_header.jsx

```diff
@@ -45,19 +50,29 @@ const usesTodayRenderer = (displayNumber, pluralReady) => (
   />
 );
 
-export const HashtagHeader = injectIntl(({ tag, intl, disabled, onClick }) => {
+export const HashtagHeader = withIdentity(injectIntl(({ tag, intl, disabled, onClick, identity }) => {
```

Comment:
It probably wouldn't hurt to switch that to using hooks rather than higher-order components here, but that could be a follow-up PR.

Reply:
Yeah, I wanted to keep the changes as minimal as possible